packages/block-library/src/quote/transforms.js

```diff
@@ -1,71 +1,150 @@
 /**
  * WordPress dependencies
  */
-import { createBlock } from '@wordpress/blocks';
-import { create, join, split, toHTMLString } from '@wordpress/rich-text';
+import {
+	createBlock,
+	parseWithAttributeSchema,
+	rawHandler,
+	serialize,
+	switchToBlockType,
+} from '@wordpress/blocks';
+
+const toBlocksOfType = ( blocks, type ) => {
+	const result = [];
+	blocks.forEach( ( block ) => {
+		if ( type === block.name ) {
+			return result.push( block );
+		}
+		const newBlocks = switchToBlockType( block, type );
+		if ( newBlocks ) {
+			result.push( ...newBlocks.filter( Boolean ) );
+		}
+	} );
+	return result;
+};
+
+const unwrapContainers = ( blocks ) =>
+	blocks.length === 1 && blocks[ 0 ].name === 'core/group'
+		? blocks[ 0 ].innerBlocks
+		: blocks;
```

Comment:
Why do group containers need to unwrap?

Reply:
Note that this only applies to single Group blocks. I think the idea was that all the same styling options on a Group would also be on the Quote, so avoiding extra wrapping elements would be desirable. But I'm going to guess that's not actually the case (especially not when themes are taken into account), so it's probably best to remove this special handling and improve the consistency here.